freezeyt/freezing.py

```diff
@@ -42,8 +42,11 @@ def freeze(app, path):
     path = Path(path)
 
     def start_response(status, headers):
-        print('status', status)
-        print('headers', headers)
+        if "200" not in status:
+            raise ValueError("Found broken page")
```

Comment:
Maybe it would be useful (or it's just my personal preference) to make this part more informative. For example printing the broken link, so that you don't need to look through the error message (but the information is currently in the log, so it isn't really necessary to put it here also) and just see it there, next to the information about finding a broken link.